oximeter/oximeter/schema/virtual-machine.toml

```diff
@@ -17,6 +17,15 @@ versions = [
     { added_in = 1, fields = [ "state", "vcpu_id" ] }
 ]
 
+[[metrics]]
+name = "vmm_vsz"
```

Comment:
somewhat on the fence about this naming, since the oximeter metrics are functionally a public user interface; I _might_ consider not abbreving `vsz` for clarity? but i might be wrong about that...